indy_common/auth_cons_strategies.py

```diff
@@ -0,0 +1,58 @@
+from abc import abstractmethod, ABCMeta
+
+from indy_common.auth_constraints import AbstractAuthConstraint
+from indy_common.auth_actions import split_action_id
+
+
+class AbstractAuthStrategy(metaclass=ABCMeta):
+    def __init__(self, auth_map):
+        self.auth_map = auth_map
+
+    @abstractmethod
+    def get_auth_constraint(self, action_id) -> AbstractAuthConstraint:
+        raise NotImplementedError()
+
+    @abstractmethod
+    def _find_auth_constraint_key(self, action_id):
+        raise NotImplementedError()
+
+    @staticmethod
+    def is_accepted_action_id(from_auth_map, from_req):
```

Comment:
Please add more tests for `is_accepted_action_id` and `LocalAuthStrategy.get_auth_constraint`

Reply:
done